front/src/components/FeedArticleCard.tsx

```diff
@@ -55,25 +63,25 @@ export default function FeedArticleCard({ feedArticle }: FeedArticle) {
     <AnimateTouchableWithoutFeedback
       onPress={() => {
         clickArticleAnimate();
-        navigation.navigate("FeedArticle");
+        navigation.navigate("FeedDetail", {
+          article_id: article_id,
+        });
```

Comment:
ES6부터 오브젝트 안에서 key와 value의 변수 명이 같을 경우 value를 생략할 수 있어요.

```suggestion
        navigation.navigate("FeedDetail", { article_id });
```

로 변경하는 것이 깔끔할것같아요.